### PR TITLE
Log both success and error counts on redis connector functions

### PR DESCRIPTION
**Pull request by tiffanycheng**

### CODE REVIEW

**Comment by junchaowu** [comment]
why both

**Comment by tiffanycheng** [comment]
turns out it's hard to get a success rate when i'm only keeping track of errors

File: connectors/redis/redis.go [hunk 1]
```diff
@@ -221,25 +221,23 @@ func (c *Connector) Remove(ctx context.Context, ei *dosa.EntityInfo, keys map[st
...
 func (c *Connector) logHitRate(method string, err error) {
-	if err != nil {
-		if dosa.ErrorIsNotFound(err) {
-			c.incStat("miss", method)
-			return
-		}
-		c.logError(method, err)
+	if err != nil && dosa.ErrorIsNotFound(err) {
+		c.incStat("miss", method)
 		return
```
Comment (jaricftw) [inline connectors/redis/redis.go]: This `return` should be removed? Otherwise we don't log the error here.

Comment (tiffanycheng) [reply connectors/redis/redis.go]: So for redis, I am trying to distinguish the case where you try to read a key but the key is not in redis from the case where you try to do a read but redis has some sort of i/o error. I guess we could increment both, but then it looks like we're double counting # of queries

**Pull request merged**